src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/cache.html.twig

```diff
@@ -0,0 +1,120 @@
+{#
+ # This file is part of php-cache\cache-bundle package.
+ #
+ # (c) 2015-2015 Aaron Scherer <aequasi@gmail.com>
+ #
+ # This source file is subject to the MIT license that is bundled
+ # with this source code in the file LICENSE.
+ #}
+
+{% extends 'WebProfilerBundle:Profiler:layout.html.twig' %}
+
+{% block head %}
+    {{ parent() }}
+    <style>
+        .color-red {
+            color:red;
+        }
+    </style>
+{% endblock head %}
+
+{% block toolbar %}
+    {% if collector.totals.calls > 0 %}
+        {% set icon %}
+            {{ include('@WebProfiler/Icon/cache.svg') }}
+            <span class="sf-toolbar-value">{{ collector.totals.calls }}</span>
+            <span class="sf-toolbar-info-piece-additional-detail">
+                <span class="sf-toolbar-label">in</span>
+                <span class="sf-toolbar-value">{{ '%0.2f'|format(collector.totals.time * 1000) }}</span>
+                <span class="sf-toolbar-label">ms</span>
+            </span>
+        {% endset %}
+        {% set text %}
+        <div class="sf-toolbar-info-piece">
+            <b>Cache Calls</b>
+            <span>{{ collector.totals.calls }}</span>
+        </div>
+        <div class="sf-toolbar-info-piece">
+            <b>Total time</b>
+            <span>{{ '%0.2f'|format(collector.totals.time * 1000) }} ms</span>
+        </div>
+        <div class="sf-toolbar-info-piece">
+            <b>Cache hits</b>
+            <span>{{ collector.totals.hits }}/{{ collector.totals.reads }} ({{ collector.totals.ratio }})</span>
+        </div>
+        <div class="sf-toolbar-info-piece">
+            <b>Cache writes</b>
+            <span>{{ collector.totals.writes }}</span>
+        </div>
+        {% endset %}
+        {% include 'WebProfilerBundle:Profiler:toolbar_item.html.twig' with { 'link': profiler_url } %}
+    {% endif %}
+{% endblock %}
+
+{% block menu %}
+    <span class="label {{ collector.totals.calls == 0 ? 'disabled' }}">
+    <span class="icon">
+        {{ include('@WebProfiler/Icon/cache.svg') }}
+    </span>
+    <strong>Cache</strong>
+    <span class="count">
+        <span>{{ collector.totals.calls }}</span>
+        <span>{{ '%0.2f'|format(collector.totals.time * 1000) }} ms</span>
+    </span>
+</span>
+{% endblock %}
+
+{% block panel %}
+    <h2>Cache</h2>
+    {% for name, calls in collector.calls %}
```

Comment:
Sorry, I misunderstood you. I put the aggregated stats for each adapter as **big numbers**. 
<img width="334" alt="screen shot 2017-01-02 at 18 46 11" src="https://cloud.githubusercontent.com/assets/1275206/21593831/d787e3c4-d11b-11e6-8636-998abf1abb85.png">